frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/result-tab/result-tab-row.scss

```diff
@@ -0,0 +1,17 @@
+.kubevirt-create-vm-modal___result-tab-row-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.kubevirt-create-vm-modal___result-tab-row-container--error > button {
+    color: #a30000 !important;
```

Comment:
You could use `var(--pf-global--danger-color--200)` here

Reply:
fixed